shared/actions/chat.js

```diff
@@ -96,6 +115,14 @@ function deleteMessage (message: Message): DeleteMessage {
   return {type: Constants.deleteMessage, payload: {message}}
 }
 
+function onAttach (conversationIDKey: ConversationIDKey, filename: string, title: string): Constants.ClickedAttach {
```

Comment:
Nice to name the method after the action type.. e.g. `function clickedAttach`.

Probably also should change `clickedAttach` to `clickedAttachment` since we don't abbreviate it elsewhere?